### ISSUE TITLE
Period added when information missing

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#499]
When metadata isn't provided, the generic statements all have a period at the end. For example, in s. 4.4.3 of the epub techniques the ELSE statement has:

> "No information about pre-recorded audio is available."

I assume these periods should be removed to match the other statements?
